jb-yoda/src/main/java/life/jukebox/JukeboxCreatedMessageReceiver.java

```diff
@@ -0,0 +1,17 @@
+package life.jukebox.yoda;
+
+import com.google.cloud.ServiceOptions;
+import com.google.cloud.pubsub.v1.AckReplyConsumer;
+import com.google.cloud.pubsub.v1.MessageReceiver;
+import com.google.cloud.pubsub.v1.Subscriber;
+import com.google.pubsub.v1.PubsubMessage;
+import com.google.pubsub.v1.SubscriptionName;
+
+public class JukeboxCreatedMessageReceiver implements MessageReceiver {
+
+  @Override
+  public void receiveMessage(PubsubMessage message, AckReplyConsumer consumer) {
```

Comment:
Please add javadoc for all public methods

Reply:
Partially done. These methods are being developed and are not final at all. It is still the "Handshake" stage.

Reply:
I'll LGTM for now - but even when something is WIP it should be documented; as the functionality develops, so will the documentation :)